samples/boards/intel_s1000_crb/audio/audio_app.rst

```diff
@@ -52,15 +60,32 @@ Pin 12 needs to be wired to Pin 29 on the 40-pin connector of the cable.
    :width: 442px
    :alt: Cable rework to play audio from host
 
-Building and Running
-********************
+Building
+********
 
 .. zephyr-app-commands::
    :zephyr-app: samples/boards/intel_s1000_crb/audio
    :board:
    :goals: build
    :compact:
 
+Running
+*******
+
+Upon power up and downloading the application image (or booting the board
+with the image in flash), the application starts without user intervention
+and audio is transferred from the microphones to the connected connected
+host and from the host to the audio codec output.
+
+If a Linux host is connected via USB to the Intel® Sue Creek S1000 board,
+the python script ``audio.py`` can be used to stop/[re]start audio transfer.
```

Comment:
Rewrite this as:
```
the python script ``audio.py`` can be used to start, stop, or restart audio transfer::

   $ cd $ZEPHYR_BASE/samples/boards/intel_s1000_crb/audio
   $ sudo -E python3 audio.py stop
   $ sudo -E python3 audio.py start

```
(Note the two colons ``::`` at the end of that line, and the indented command line examples)

Reply:
used `code-block` instead. re-check.

Reply:
That works, thanks.